### PR TITLE
`Alignment/OfflineValidation` improvements from CRUZET'21 data analysis

### PR DESCRIPTION
**Pull request by mmusich**
#### PR description:

Introduce some miscelleaneous improvements in the `EDAnalyzers` used to cross-check the Cosmics datasets for tracker alignment purposes, mostly  adding a Pixel on-track clusters occupancy map per ROC.  
I profit of this PR to introduce some other changes in the `DQM/TrackerRemapper` are added in order to be able display empty maps with marked singled out modules and ROCs. 

#### PR validation:

Private tests.

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

N/A

### CODE REVIEW

File: Alignment/OfflineValidation/plugins/DMRChecker.cc [hunk 1]
```diff
@@ -630,7 +630,7 @@ class DMRChecker : public edm::one::EDAnalyzer<edm::one::SharedResources> {
             uOrientation = deltaPhi(gUDirection.barePhi(), gPModule.barePhi()) >= 0. ? +1.F : -1.F;
             //vOrientation = gVDirection.z() - gPModule.z() >= 0 ? +1.F : -1.F; // not used for Strips
```
Comment (tvami) [inline Alignment/OfflineValidation/plugins/DMRChecker.cc:631]: Please remove commented out code

File: Alignment/OfflineValidation/plugins/DMRChecker.cc [hunk 2]
```diff
@@ -647,7 +647,7 @@ class DMRChecker : public edm::one::EDAnalyzer<edm::one::SharedResources> {
             uOrientation = deltaPhi(gUDirection.barePhi(), gPModule.barePhi()) >= 0. ? +1.F : -1.F;
             //vOrientation = gVDirection.perp() - gPModule.perp() >= 0. ? +1.F : -1.F; // not used for Strips
```
Comment (tvami) [inline Alignment/OfflineValidation/plugins/DMRChecker.cc:648]: Please remove commented out code

File: Alignment/OfflineValidation/plugins/DMRChecker.cc [hunk 3]
```diff
@@ -657,7 +657,7 @@ class DMRChecker : public edm::one::EDAnalyzer<edm::one::SharedResources> {
...
```
Comment (tvami) [inline Alignment/OfflineValidation/plugins/DMRChecker.cc:658]: Please remove commented out code

File: Alignment/OfflineValidation/plugins/DMRChecker.cc (hunk 2 above)
Comment (mmusich) [reply Alignment/OfflineValidation/plugins/DMRChecker.cc:648]: nope.

File: Alignment/OfflineValidation/plugins/DMRChecker.cc (hunk 3 above)
Comment (mmusich) [reply Alignment/OfflineValidation/plugins/DMRChecker.cc:658]: nope.

**Comment by mmusich** [comment]
@cmsbuild, please test

File: Alignment/OfflineValidation/plugins/DMRChecker.cc (hunk 1 above)
Comment (mmusich) [reply Alignment/OfflineValidation/plugins/DMRChecker.cc:631]: nope.

**Comment by tvami** [comment]
Hi @mmusich the SA is saying that `Deprecated API | EventSetupRecord::get function called | /Alignment/OfflineValidation/plugins/TrackerOfflineValidationSummary.cc` should this be taken care in a separate PR?

**Comment by mmusich** [comment]
> he SA is saying that Deprecated API | EventSetupRecord::get function called | /Alignment/OfflineValidation/plugins/TrackerOfflineValidationSummary.cc should this be taken care in a separate PR?

clearly not part of this PR.

**Comment by tvami** [comment]
+alca 

 * Diffs in the comparison are due to "The workflows 140.53 have different files in step1_dasquery.log than the ones found in the baseline" otherwise all good
 * SA msg will be taken care in a separate PR

**Comment by jfernan2** [comment]
+1

**Comment by perrotta** [comment]
+1

**Pull request merged**